GitHubContributionsWidget/Extensions/ThemeExtensions.swift

```diff
@@ -26,6 +26,8 @@ extension Theme {
         case .blue: return .blueLevel1
         case .red: return .redLevel1
         case .purple: return .purpleLevel1
+        case .yellow: return .yellowLevel1
+        case .invert: return .invertLevel1
```

Comment:
what do you think about renaming the "Invert" them to "Inverted"?

Reply:
fixed at https://github.com/AnderGoig/github-contributions-ios/pull/20/commits/64cd790b0406ce2c37c65d610546c3d0ce1ae2ce